src/raw/config-avalanche.js

```diff
@@ -132,22 +156,38 @@ config = {
     },
     {
       index: 3,
-      name: "QI Market",
+      name: "QI",
       description:
         "QI is the governance token for BENQI, Avalanche's leading liquidity protocol. QI is earned as a reward for supplying liquidity in BENQI, and can be supplied in their protocol for a tasty APY.",
-      oracleHeartbeatInSec: 1800,
-      icon: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/qi/qi.png",
-      oracleDecimals: 8,
+      iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/tokens/qi/qi.png",
       leverage: 2,
-      paymentTokenSymbol: "DAI",
+      paymentToken: {
+        address: "0xd586e7f844cea2f87f50152665bcbc2c279d8d70",
+        isUSDBased: true,
+        decimals: 18,
+        symbol: "DAI",
+        iconUrl: "https://media-float-capital.fra1.cdn.digitaloceanspaces.com/public/icons/tokens/dai.svg",
+      },
+      unitOfMeasure: "$",
+      isUnitSuffix: false,
       symbol: "2QI",
-      launchTimestamp: 1642514400,
+      plannedLaunchTimestamp: 1642514400,
```

Comment:
Whats the destinction between launchTimestamp and plannedLaunchTimestamp, is this because its actually just when the ui updates to show the market?

Reply:
The planned one is made ahead of time, and it's how we show countdowns to new markets. The actual launch timestamp (I'll change the name to `actualLaunchTimestamp` to make it more clear) is when the market initialize function on LongShort is called